tests/test_postprocess.py

```diff
@@ -92,6 +97,19 @@ def test_decode_times(times):
     assert dsa['Time'].attrs['standard_name'] == 'time'
 
 
+@pytest.mark.parametrize(
+    'sample_dataset_with_kwargs,xtime_dtype',
+    [
+        (('mercator', {'decode_times': True}), 'timedelta64[ns]'),
+        (('mercator', {'decode_times': False}), 'float32'),
+    ],
+    indirect=['sample_dataset_with_kwargs'],
+)
+def test_xtime_handling(sample_dataset_with_kwargs, xtime_dtype):
+    dataset = xwrf.postprocess._decode_times(sample_dataset_with_kwargs)
+    assert dataset.XTIME.dtype == xtime_dtype
```

Comment:
I think it's more common to use `assert np.issubtype(dataset.XTIME.dtype, np.timedelta64)` or something like that.

Reply:
You are probably right. I was trying to keep it in line with L95.

Reply:
Changed it in 2cdab9a :)